packages/tools/src/tools/annotation/PlanarFreehandROITool.ts

```diff
@@ -191,6 +191,8 @@ class PlanarFreehandROITool extends ContourSegmentationBaseTool {
           radius: 2,
         },
         allowOpenContours: true,
+        // Allow combining by clicking outside/dragging into this contour
+        allowCombine: true,
```

Comment:
not sure what is this

Reply:
This can be removed after adding something like an `if(contourSegUtils.isInterpolatedContourAnnotation(annotation)) return;` in `contourSegmentationCompleted.ts`